pkg/network/usm/config/config.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+// Package config provides helpers for USM configuration
+package config
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/DataDog/datadog-agent/pkg/network/protocols/http"
+)
+
+// NeedUSM returns true if USM is supported and enabled
+func NeedUSM(config *config.Config) bool {
```

Comment:
nit: consider renaming the function to better convey its purpose of checking support and enabling status (e.g., isUSMSupportedAndEnabled)

Reply:
Done.